lib/js/components/Buttons/Button/Button.stories.ts

```diff
@@ -0,0 +1,104 @@
+// Button.stories.ts
+
+import Button from './Button.vue';
+import {
+	BUTTON_SIZES,
+	BUTTON_TYPES,
+	BUTTON_COLORS,
+	BUTTON_STATES,
+	BUTTON_RADIUSES,
+	BUTTON_ELEVATIONS,
+} from './Button.consts';
+import { ICONS } from '../../Icon';
+
+import { Meta, Story } from '@storybook/vue';
+
+export default {
+	title: 'Components/Buttons/Button',
+	component: Button,
+} as Meta<typeof Button>;
+
+const Template: Story<{
+	slotText: Object;
+	size: Object;
+	type: Object;
+	color: Object;
+	radius: Object;
+	state: Object;
+	elevation: Object;
+	iconLeft: Object;
+	iconRight: Object;
+}> = (argTypes) => ({
+	components: { Button },
+	props: Object.keys(argTypes),
+	template: `<div :class="{ contrastBackground: isInverted }">
+                <Button
+                  :size="size"
+                  :type="type"
+                  :state="state"
+                  :radius="radius"
+                  :color="color"
+                  :elevation="elevation"
```

Comment:
did you consider using `v-bind=$props` syntax? I'm not sure what are dowsnsides of it, but it looks promising

Reply:
I think this is unacceptable. I will try find out other way.

Reply:
Btw. This icon in `Badge` I think we can put for now like this:
```
src="https://lek.wiecejnizlek.pl/images/lek/logo-badge.svg" 
```
this will be replaced anyway in Badge refactoring tomorrow.

Reply:
@kvas-damian I found a way, but we need discuss is it profitable to keep such a code:
```
template: `<div :class="{ contrastBackground: isInverted }">
                <Button
			v-bind="propsWithoutSlot"
                  	:icon-left="ICONS[iconLeft]"
                  	:icon-right="ICONS[iconRight]"
                >
                    {{slotText}}
                </Button>
            </div>`,
	computed: {
		propsWithoutSlot() {
			const all = Object.assign({}, this.$props);
			delete all.slotText
			return all;
		}
	},
```

Reply:
it's not worth it. We can stay with the old version of passing props